modules/placement-driver/src/main/java/org/apache/ignite/internal/raft/client/TopologyAwareRaftGroupService.java

```diff
@@ -418,10 +431,8 @@ private static class ServerEventHandler {
          * @param term Term.
          */
         private synchronized void onLeaderElected(ClusterNode node, long term) {
-            if (onLeaderElectedCallback != null && term > this.term) {
```

Comment:
Why does the expression (term > this.term) is removed?
There is no sense to notify several times about one term.